### ISSUE TITLE
Sending 

### ISSUE THREAD

**Issue by dimus** [xyproto/wallutils#33]
Hello @xyproto, I want to make a more [interesting backgrounds for proportionally scaled images](https://codeberg.org/dimus/muro). I use wallutils as a backend.

To achieve this I create a wallpaper of the right proportions and then use 'fill' method to display it. All works well for one monitor, but if there are several monitors of different dimensions, 'fill' method breaks.

It would be great to have an ability to send an individually adjusted image to each monitor. Would it it be possible?

**Comment by xyproto** [comment]
Yes, it should be possible, for most window managers, I believe. Thanks for the feature request!

**Comment by dimus** [comment]
Here is a relevant answer on how such functionality can be achieved with feh: https://stackoverflow.com/a/74822443/23080
